README.md

```diff
@@ -1,10 +1,45 @@
+[![Build Status](https://badge.buildkite.com/9e0e6c88972a3248a0908506d6946624da84e4e18c0870c4d0.svg)](https://buildkite.com/rust-vmm/kvm-ioctls-ci)
+![crates.io](https://img.shields.io/crates/v/$CRATE.svg)
+
 # kvm-ioctls
 
-TODO
+The kvm-ioctls crate provides safe wrappers over the KVM API, a set of ioctls
+used for creating and configuring Virtual Machines (VMs). The ioctls are
+accessible through four structures:
+- Kvm - wrappers over system ioctls
+- VmFd - wrappers over VM ioctls
+- VcpuFd - wrappers over vCPU ioctls
+- DeviceFd - wrappers over device ioctls
+
+For further details check the
+[KVM API](https://www.kernel.org/doc/Documentation/virtual/kvm/api.txt) as well
+as the code documentation.
+
+## Supported Platforms
+
+The kvm-ioctls can be used on x86_64 and aarch64. Right now the aarch64 support
+is consider experimental. For a production ready version, please check the
+progress in the corresponding
+[GitHub issue](https://github.com/rust-vmm/kvm-ioctls/issues/8).
 
 ## Running the tests
 
-TODO
+Our Continuous Integration (CI) pipeline is implemented on top of Buildkite.
```

Comment:
Maybe add a link to the Buildkite website? New contributors might not be familiar with it.